common.blocks/menu/menu.bemhtml

```diff
@@ -38,13 +38,23 @@ block('menu')(
             _checkedItems : checkedItems,
             _menuMods : {
                 theme : mods.theme,
-                disabled : mods.disabled
+                disabled : mods.disabled,
+                mode : mods.mode
             }
         });
     }),
     attrs()(function() {
-        var attrs = { role : 'menu' };
-        this.mods.disabled || (attrs.tabindex = 0);
+        var ctx = this.ctx,
+            mods = this.mods,
+            attrs = {
+                role : 'menu',
+                'aria-disabled' : mods.disabled
+            };
+
+        mods.disabled?
+            attrs['aria-disabled'] = true :
```

Comment:
Do you set `aria-disabled` attribute twice? It seems that something wrong with logic.